### DISCUSSION TITLE
Official WG meeting

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#115] [Ideas]
I'd like to try and formally bring back the official WG meeting. I want it to be low overhead so that it's hopefully sustainable. I think it would be best run on Zoom, since it seems like it would be a good generic option that would support a lot of people. Note, I don't have a paid account, so we would be limited to 40 minutes. Wouldn't be the worst thing though.

If there was interest, we might be able to ask the Rust Foundation to host a Jitsi instance for us. But I imagine we couldn't get that going right away. I personally would prefer to use Discord, but I know that's not everyone's favorite option.

## Schedule

I want to try and aim for about every two weeks, with the cadence being "first Tuesday of the month", and "third Tuesday of the month". I'm picking Tuesday as it's the day I'm most free. I sent out a Doodle Poll last year, but it got deleted somehow ://

I aim to have both of them happening at two separate time slots:

First Tuesday (Asia friendly time, 08:00 EST, my time): https://everytimezone.com/s/430f61be
Third Tuesday (NA/EU friendly time, 14:00 EST, my time): https://everytimezone.com/s/5a53af68

I'm loosely basing this off the [CTCFT schedule](https://rust-ctcft.github.io/ctcft/meetings.html#timing-of-the-meeting), but aiming for earlier in the NA day.

## Agenda

Here are some items that I imagine being at these meetings. However, I would like more suggestions for what they could generally include.

### First Tuesday

- Effort to aggregate topics for the newsletter, call to action to write sections

### Third Tuesday

- Discuss a larger WG issue each month, dive into the weeds a bit

## Disclaimer

I'm writing a lot in the form of "this is how it will be", but that's just because I want to make it happen and not get too caught up in pre-planning it :slightly_smiling_face: once it has run a few times, I think we can certainly improve the workflow. I am certainly open to feedback though.

**Comment by AngelOnFira** [comment]
@Lokathor I'd be interested on your feedback, it seems you took many notes from the previous meetups :+1:

**Comment by ozkriff** [comment]
It'd be great to have regular gamedev WG calls in any form again! Not sure I'm personally will be able to often participate on Tuesdays though - are the calls supposed to be recorded?

> I want it to be low overhead so that it's hopefully sustainable.

Yeah, judging by our previous failed attempts at regular meetups - it's much better to keep it as simple as possible.

**Comment by Lokathor** [comment]
My feedback is that the previous meetings fizzled out for low ability to get anything done. So really you'll want to have clear and actionable stuff in mind to accomplish during meetings.

Also I could maybe do the afternoon time for Monday, but not either of them Tuesday.

**Comment by Wodann** [comment]
I could attend the third Tuesday (at 14:00 EST) if enough people are willing to attend. In the past it was usually the same 3 or 4 people, but we didn't have enough bandwidth alongside our own open-source projects to make anything actionable.

**Comment by AlexEne** [comment]
Having clear action items is, in my opinion, quite tricky to be sustainable in the long run. 
People already likely work on passion projects. Having action items that go into that passion project time is a way to quickly get to burnout :). 
That's why I think the current format where people meet up to show off their side-project works really well.

**Reply by Wodann** [reply]
> That's why I think the current format where people meet up to show off their side-project works really well.

That's what is already being done in the Rust GameDev Discord meetup, right? Do we need to do the same with a WG meeting?

**Reply by AlexEne** [reply]
Yes, that's what the discord meetup is. I guess it depends on the intended outcome.
